From 0b0be8235ea997c14f4a2e1f3bf4f82d38834c07 Mon Sep 17 00:00:00 2001
From: Steffen Sledz <sledz@dresearch-fe.de>
Date: Wed, 1 Jun 2016 13:03:45 +0200
Subject: [PATCH] libahci_platform: add missing symbol export

This fixes errors like this one:
ERROR: "ahci_platform_sht" [drivers/ata/ahci_imx.ko] undefined!

Signed-off-by: Steffen Sledz <sledz@dresearch-fe.de>
---
 drivers/ata/libahci_platform.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/ata/libahci_platform.c b/drivers/ata/libahci_platform.c
index e26decd..0c4fd83 100644
--- a/drivers/ata/libahci_platform.c
+++ b/drivers/ata/libahci_platform.c
@@ -37,6 +37,7 @@ EXPORT_SYMBOL_GPL(ahci_platform_ops);
 struct scsi_host_template ahci_platform_sht = {
 	AHCI_SHT("ahci_platform"),
 };
+EXPORT_SYMBOL_GPL(ahci_platform_sht);
 
 /**
  * ahci_platform_enable_clks - Enable platform clocks
-- 
2.8.3

